revscoring/utilities/test.py

```diff
@@ -0,0 +1,116 @@
+"""
+``revscoring test -h``
+::
+
+    Tests a scorer model.  This utility expects to get a file of
+    tab-separated feature values and labels from which to test a model.
+
+    Usage:
+        train_test -h | --help
+        train_test <scorer_model> <features> [-p=<kv>]... [-s=<kv>]...
+                   [--version=<vers>]
+                   [--values-labels=<path>]
+                   [--model-file=<path>]
+                   [--label-type=<type>]
+                   [--scale]
+                   [--debug]
+
+    Options:
+        -h --help               Prints this documentation
+        <scorer_model>          Classpath to an the MLScorerModel to construct
```

Comment:
This should be a path to a model file that has already been trained -- not the scorer model class.